openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -202,3 +205,23 @@ def partial_update(self, request, username):
             )
 
         return Response(account_settings)
+
+
+class AccountDeactivationView(APIView):
+    """
+    Account deactivation viewset. Currently only supports POST requests.
+    Only admins can deactivate accounts.
+    """
+    permission_classes = (permissions.IsAuthenticated, IsStaffUser)
+
+    def post(self, request, username):
+        """
+        POST /api/user/v1/accounts/{username}/deactivate/
+
+        Changes the 'is_active' attribute of a user.
+        """
+        user = User.objects.get(username=username)
+        user.is_active = False
```

Comment:
Sadly, this doesn't mean what you and I think it means. Someone unwisely overloaded this bit to indicate if the user has verified his/her email address after account creation.